Mergin/plugin.py

```diff
@@ -262,6 +265,46 @@ def configure(self):
             self.on_config_changed()
             self.show_browser_panel()
 
+    def show_no_workspaces_dialog(self):
+        msg = (
+            "Workspace is a place to store your projects and share them with your colleagues. "
+            "Click on the button below to create one. \n\n"
+            "A minimum of one workspace is required to use Mergin Maps."
+        )
+        msg_box = QMessageBox(QMessageBox.Critical, "You do not have any workspace", msg, QMessageBox.Close)
+        create_button = msg_box.addButton("Create workspace", msg_box.ActionRole)
+        create_button.clicked.disconnect()
+        create_button.clicked.connect(self.open_configured_url)
+        msg_box.exec_()
+
+    def set_current_workspace(self, workspace):
+        settings = QSettings()
+        self.current_workspace = workspace
+        settings.setValue("Mergin/lastUsedWorkspace", str(workspace))
```

Comment:
is not is safer to use workspace_id? Just in case we allow renaming in future. Or workspace is removed and recreated again.

Reply:
Yes, that's a good point

Reply:
fixed